qendpoint-store/src/main/java/com/the_qa_company/qendpoint/compiler/SailCompiler.java

```diff
@@ -400,8 +400,7 @@ public LinkedSail<? extends NotifyingSail> compileNode(Value node) throws SailCo
 		 */
 		public Value searchOne(Resource subject, IRI predicate) throws SailCompilerException {
 			Value out;
-			try (CloseableIteration<? extends Statement, SailCompilerException> it = connection.getStatements(subject,
-					predicate, null)) {
+			try (CloseableIteration<? extends Statement> it = connection.getStatements(subject, predicate, null)) {
```

Comment:
In RDF4J 5.0.0 the CloseableIteration interface and all other iterators/iterations no longer support generic declaration of exceptions. You can still throw an exception, but it can no longer be declared and thus it needs to be a RuntimeException.